### PR TITLE
Added support for docker-compose command

### PR DESCRIPTION
**Pull request by dimalmfao**
for some reason, it added my previous commits. the [last 2](https://github.com/0-5788719150923125/vtx/pull/25/commits).

### CODE REVIEW

**Comment by prisect** [comment]
fuck

ok better now

File: controller.sh [hunk 1]
```diff
@@ -10,8 +19,8 @@ if ! command -v docker &> /dev/null; then
 fi
 
 # Check for docker-compose
-if ! command -v docker compose &> /dev/null; then
```
Comment (Vectorrent) [inline controller.sh:13]: This looks pretty good, but go ahead and remove these checks here. There's no reason to check for the existence of docker compose twice!

File: controller.sh [hunk 2]
```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 
-CONTAINERS='["lab", "ctx", "uxo", "tbd", "ipf", "pet", "bit"]'
```
Comment (Vectorrent) [inline controller.sh:3]: Why did you delete this variable?

File: controller.sh (hunk 1 above)
Comment (prisect) [reply controller.sh:13]: didnt see that sorry

File: controller.sh (hunk 2 above)
Comment (prisect) [reply controller.sh:3]: what? i didnt delete anything

Comment (Vectorrent) [reply controller.sh:3]: This red section says otherwise :smile: 

Just add that line back, and I think we'll be good to merge the PR. Thanks!

**Comment by prisect** [comment]
placed models at line 3

**Comment by Vectorrent** [comment]
lgtm, thank you!
